static/app/components/events/interfaces/performance/utils.tsx

```diff
@@ -32,6 +32,9 @@ const RESOURCES_DESCRIPTIONS: Record<IssueType, string> = {
   [IssueType.PERFORMANCE_SLOW_SPAN]: t(
     'Slow DB Queries are SELECT query spans that take longer than 1s. A quick method to understand why this may be the case is running an EXPLAIN command on the query itself. To learn more about how to fix slow DB queries, check out these resources:'
   ),
+  [IssueType.PERFORMANCE_UNCOMPRESSED_ASSET]: t(
+    'Uncompressed assets are assets that take over 50ms and are larger than 512kB which can usually be made faster with compression. Check that your server or CDN serving your assets is accepting the content encoding header from the browser and is returning them compressed.'
```

Comment:
I think this is a tad more accurate
```suggestion
    'Uncompressed assets are asset spans that take over 50ms and are larger than 512kB which can usually be made faster with compression. Check that your server or CDN serving your assets is accepting the content encoding header from the browser and is returning them compressed.'
```